layouts/partials/hero/basic.html

```diff
@@ -0,0 +1,24 @@
+{{ $disableImageOptimization := .Page.Site.Params.disableImageOptimization | default false }}
+
+{{- $images := .Resources.ByType "image" -}}
+{{- $featured := $images.GetMatch "*feature*" -}}
+{{- if not $featured }}{{ $featured = $images.GetMatch "{*cover*,*thumbnail*}" }}{{ end -}}
+{{ if .Params.featureimage }}
+{{- $url:= .Params.featureimage -}}
+{{- if not $featured }}{{ $featured = resources.GetRemote $url }}{{ end -}}
+{{ end }}
+{{ if .Params.featureAsset }}
+    {{- $genericImage := .Params.featureAsset -}}
+    {{- if not $featured }}{{ $featured = resources.Get $genericImage }}{{ end -}}
+{{ end }}
```

Comment:
Only these lines have been added.

This file is a copy of `themes/blowfish/layouts/partials/hero/basic.html` with only the highlighted lines (10-13) added to override the theme's standard file.